Makefile

```diff
@@ -1146,13 +1146,17 @@ version: $(VERSRC)
 $(VERSRC): Makefile
 	VERSION=$(VERSION) $(MAKE) -f version.mk setver
 
-# make tag - prints a tag to use with git for the current version
-# 	To put a new release on Github:
-# 		- bump VERSION variable
-# 		- run make setver
-# 		- commit changes to git
-# 		- build binaries with 'make release'
-# 		- run `make tag` and use its output to 'git tag' and 'git push --tags'
+# Pushes GITTAG to GitHub and builds the corresponding release.
```

Comment:
nit: technically, this target doesn't build any corresponding release - it just pushes a tag. What happens after that is not part of this make target.

Reply:
Well spotted, done.